### ISSUE TITLE
Support for other languages?

### ISSUE THREAD

**Issue by jsve** [Jason-Rev/vscode-spell-checker#123]
What are your plans for this? I'm looking for a solution to spellcheck markdown and latex files with Swedish text, but without luck. 

Swedish spellchecking is kind of available in some projects but not for latex files. Spellchecking latex files with your plugin is reportedly working, but - as you know - without language support.

**Comment by Jason3S** [comment]
Please open an issue on [cspell-dicts](https://github.com/Jason3S/cspell-dicts/issues).

If you have a preferred hunspell library, then please include a link. I have been using this repository for hunspell files: [titoBouzout/Dictionaries](https://github.com/titoBouzout/Dictionaries)

**Comment by Jason3S** [comment]
I went ahead and added Swedish. You can install it from here: [npm cspell-dict-sv](https://www.npmjs.com/package/cspell-dict-sv).

You will need to enable checking Swedish in your VS Code settings. For "cSpell.language" change it to:
```
"cSpell.language":"en,sv"
```

**Comment by Jason3S** [comment]
To make installing Swedish easier, I created an add-on extension:
https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker-swedish

**Comment by Jason3S** [comment]
I'm going to close this issue. Please let me know if the extension works for you.
